src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -889,8 +871,22 @@ let cmdline_term ~allow_empty_config : conf Term.t =
       | _ when gitlab_sast -> Output_format.Gitlab_sast
       | _ when gitlab_secrets -> Output_format.Gitlab_secrets
       | _ when junit_xml -> Output_format.Junit_xml
-      | _else_ -> default.output_format
+      | _else_ -> default.output_conf.output_format
     in
+    let output_conf : Output.conf =
+      {
+        nosem;
+        autofix;
```

Comment:
not quite sure how we made the cut between scan cli conf and output conf, but semantically `autofix` and `dryrun` seem more like behavioral flags vs output specifiers

Reply:
The cut was "what's required by Output.ml". I also agree, but I also don't want to duplicate code if possible